kvmagent/kvmagent/test/shareblock_testsuite/test_sblk_metadata.py

```
@@ -0,0 +1,1006 @@
+        start_event = threading.Event()
+
+        def writer(payload_val):
+            try:
+                start_event.wait(30)
+                h = self._make_handler()
+                h._do_write(
+                    meta_path, '{"t":%d}' % payload_val,
+                    vmUuid=vm_uuid, vmName='',
+                    vmCategory='', architecture='',
+                    schemaVersion='')
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)